<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RockImageUI::RockImageUI</class>
 <widget class="QMainWindow" name="RockImageUI::RockImageUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RockImage</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QMdiArea" name="openImagesArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Arquivos</string>
    </property>
    <addaction name="openImageAction"/>
    <addaction name="saveDataAction"/>
    <addaction name="cleanTableAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="imageMenu">
    <property name="title">
     <string>Imagem</string>
    </property>
    <widget class="QMenu" name="menuPincel">
     <property name="title">
      <string>Pincel</string>
     </property>
     <addaction name="increaseWidthAction"/>
     <addaction name="decreaseWidthAction"/>
     <addaction name="chooseColorAction"/>
    </widget>
    <addaction name="applyBinarizationAction"/>
    <addaction name="addLayerAction"/>
    <addaction name="removeLayerAction"/>
    <addaction name="menuPincel"/>
   </widget>
   <widget class="QMenu" name="showDockMenu">
    <property name="title">
     <string>Exibir</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="imageMenu"/>
   <addaction name="showDockMenu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="imagesListDockWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Lista de Imagens</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="imagesListDockWidgetContent">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QListWidget" name="imagesList"/>
     </item>
     <item row="2" column="0">
      <widget class="QTreeWidget" name="imageTree">
       <column>
        <property name="text">
         <string>Nome</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cor</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="tableTabDockWidget">
   <property name="windowTitle">
    <string>Tabelas de Dados</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="tableTabDockWidgetContent">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QTabWidget" name="dataTablesTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="openImageAction">
   <property name="text">
    <string>Abrir Imagem</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveDataAction">
   <property name="text">
    <string>Salvar Dados da Tabela</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="cleanTableAction">
   <property name="text">
    <string>Limpar Tabela</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="applyBinarizationAction">
   <property name="text">
    <string>Aplicar Binarização</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="collectDataAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Coletar Dados</string>
   </property>
   <property name="toolTip">
    <string>Envia dados coletados para a tabela</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="changeLabelAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alterar Label</string>
   </property>
   <property name="toolTip">
    <string>Altera a label atribuida ao dado coletado</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+;</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="text">
    <string>Aproximar</string>
   </property>
   <property name="toolTip">
    <string>Aproximar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="text">
    <string>Afastar</string>
   </property>
   <property name="toolTip">
    <string>Afastar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>Fechar Todas Janelas</string>
   </property>
   <property name="toolTip">
    <string>Fecha todas as janelas na área de trabalho</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="showDataTableAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tabela de Dados</string>
   </property>
  </action>
  <action name="showImageListAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Lista de Imagens</string>
   </property>
  </action>
  <action name="addLayerAction">
   <property name="text">
    <string>Adicionar Camada</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="removeLayerAction">
   <property name="text">
    <string>Remover Camanda</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="increaseWidthAction">
   <property name="text">
    <string>Aumentar Largura</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+[</string>
   </property>
  </action>
  <action name="decreaseWidthAction">
   <property name="text">
    <string>Diminuir Largura</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="chooseColorAction">
   <property name="text">
    <string>Escolher Color</string>
   </property>
  </action>
  <zorder>tableTabDockWidget</zorder>
  <zorder>imagesListDockWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
